data/items/items.xml

```diff
@@ -17590,6 +17590,7 @@
 	</item>
 	<item id="9975" article="a" name="crystal table">
 		<attribute key="weight" value="2500" />
+		<attribute key="wrapTo" value="9974" />
```

Comment:
yes, rework it, wrap should be an itemType not an attribute, also your changes should replace constructionskits.lua in actions, delete it from your repository.